input/fsh/Example-Dataset-DD.fsh

```diff
@@ -0,0 +1,126 @@
+/**
+ All related instances necessary to show how to build a data-dictionary for a 
+ small dataset
+ */
+
+/**
+ Define the Terminology components specific to the dataset itself. 
+
+ One CodeSystem for the dataset listing each of the tables as individual codes
+
+ One CodeSystem for each of the tables with individual codes for each variable 
+ to be represented in FHIR
+
+ For Categorical variables, we'll also have a CodeSystem for each unique set of 
+ values. These shouldn't displace the need for mapping appropriately to public
+ ontologies for the sake of interoperability, however, these values should be
+ used in Codings alongside the harmonize codes for traceability. 
+*/
+CodeSystem: ExampleStudyDDStudyDatasetCS1
+Id: example-study-dd-dataset-codesystem-1
+Title: "Dataset CodeSystem (lists all tables)"
+Description: "Example data-dictionary dataset with 1 table and 4 variables"
+* ^url = $ExampleStudyDatasetCS
+* ^content = #complete
+* ^count = 1
+* #demographics
+
+CodeSystem: ExampleStudyDDStudyTableCS1
+Id: example-study-dd-datatable-codesystem-1
+Title: "Datatable CodeSystem (lists all variables)"
+Description: "Example data-dictionary data table with 4 variables"
+* ^url = $ExampleStudyTableCS
+* ^content = #complete
+* ^count = 5
+* #subjectid "Subject's Unique Identifier"
```

Comment:
This surprised me.   Reading along so far, I expected aggregations not individual observations ?  (Maybe I missed something)